### PR TITLE
added Sensitivity slider

### PR DESCRIPTION
**Pull request by Uriyel**
Only a couple line changes.  Should be self explanatory.  Wasn't sure setting the icd to global was the way to go, but worked out for me.

### CODE REVIEW

File: CollisionAlert.lua [hunk 1]
```diff
@@ -14,9 +16,9 @@ function Instance:onInit()
 		self.collisionModel = getEditor():createNew(self:getParent():getObjectKit(), "CollisionModel")	
 	end
 
-	local icd = getEditor():createNew(self.collisionModel:getCollisionDetectors(), "ImpactCollisionDetector")
+	icd = getEditor():createNew(self.collisionModel:getCollisionDetectors(), "ImpactCollisionDetector")
```
Comment (logicplace) [inline CollisionAlert.lua]: Would prefer self.icd to match the other styling (in each place it's used)

File: CollisionAlert.lua [hunk 2]
```diff
@@ -1,10 +1,12 @@
 
 Instance.properties = properties({
+	{ name="Sensitivity", type="Real", value=0, range={min=0, max=3}, onUpdate="onInstanceUpdate" },
```
Comment (logicplace) [inline CollisionAlert.lua]: Since this is a real it should have value and min/max as reals, so 0.0 and 3.0

**Review by logicplace — CHANGES_REQUESTED** [review]
Yeah it being an instance var is fine. I just have some style notes

File: CollisionAlert.lua (hunk 2 above)
Comment (Uriyel) [reply CollisionAlert.lua]: Yep.  I hadn't looked back at the properties after I set the type.

File: CollisionAlert.lua (hunk 1 above)
Comment (Uriyel) [reply CollisionAlert.lua]: I wondered about this.  I thought what if there were multiple objects in the scene using the mod.  Each should be instanced throughout yea?

Comment (logicplace) [reply CollisionAlert.lua]: Yeah it will be either way, I think

**Review by logicplace — APPROVED** [review]

**Pull request merged**